### PR TITLE
Add a Linux/AppImage build

### LINKED ISSUE #35: Official Linux build
Would be awesome to have official binaries for Linux, i.e. something like a binary tarball that ships a version of Electron and an asar archive and/or an AppImage.

A Flatpak package might be nice too.

### PR DESCRIPTION
**Pull request by fwcd**
### Fixes #35

This adds an experimental Linux build along with a corresponding Ubuntu CI configuration. The `package.json` already includes an electron-builder configuration producing an AppImage on Linux, so the changes required to the code are minimal.

### CODE REVIEW

File: src/renderer/menu/MenuBar.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import styled from 'styled-components'
 import zIndexes from '../zIndexes'
 import UniverseIcon from '@mui/icons-material/Settings'
-import LightingIcon from '@mui/icons-material/LightBulb'
+import LightingIcon from '@mui/icons-material/Lightbulb'
```
Comment (fwcd) [inline src/renderer/menu/MenuBar.tsx:5]: A minor issue that surfaced during the build were a few miscapitalized imports, which presumably weren't a problem on macOS and Windows where file systems are usually case-insensitive (as opposed to Linux).

**Comment by fwcd** [comment]
Any chance you could release a version with a Linux build on the website? The CI builds unfortunately expire regularly, which makes it a bit of a hassle to quickly grab a build for DJing if there's a Linux laptop...

By the way, I don't know of any other free/open-source lighting app that comes close to Captivate in terms of simplicity and elegance, thanks again for all of the great work!

**Comment by TheJuiceFR** [comment]
> Any chance you could release a version with a Linux build on the website? 

That would mean that it would be a supported version, even if they said it was "beta" or "unstable". I'm guessing Linux as an OS is out of scope for them.

**Comment by fwcd** [comment]
The maintenance overhead would be practically zero. It's an Electron app, the CI pipeline already produces Linux builds and I have in fact successfully used one of these for lighting at a recent gig.

As a workaround, I have now set up a release pipeline in my fork for now: https://github.com/fwcd/captivate/releases

**Pull request merged**